src/scripts/main.js

```diff
@@ -1,3 +1,59 @@
 'use strict';
 
 // write code here
+
+const employees = document.querySelectorAll('li');
+const listEmployees = document.querySelector('ul');
+
+function getSalary(element) {
+  const salary = element.dataset.salary;
+
+  if (!salary) {
+    throw new Error(`Salary not found for employee: ${element.textContent}`);
+  }
+
+  const cleanedSalary = salary.substring(1).replace(',', '');
+  const numericSalary = Number(cleanedSalary);
+
+  if (isNaN(numericSalary)) {
+    throw new Error(`Invalid salary format for employee:
+      ${element.textContent}`);
+  }
+
+  return numericSalary;
+}
+
+function sortList(list) {
+  const listItem = Array.from(list);
```

Comment:
You are converting `list` to an array, but in the next line, you are using `slice()` method which is unnecessary because `Array.from()` already returns a new array. So, you can remove the `slice()` method in the next line.